[DOC] JSON command edits (#846)
RedisJSON/RedisJSON: docs/commands/json.arrinsert.md

> Insert the `json` values into the array at `path` before the `index` (shifts to the right)

Author: is that not expected?

> Insert the `json` values into the array at `path` before the `index` (shifts to the right)

Reviewer: I assume most users would expect that, but as far as I am concerned, if we can be formal about indices being zero-based, why not?

> Insert the `json` values into the array at `path` before the `index` (shifts to the right)

Author: sure, will add.

[Examples](#examples)

## Required arguments

<details open><summary><code>key</code></summary> 

is key to modify.
</details>

<details open><summary><code>value</code></summary> 

> is one or more values to append to one or more arrays. 

Author: it should be insert, yes. copy pasta! i'll fix it

> is one or more values to append to one or more arrays. 

Author: ```suggestion
is one or more values to insert in one or more arrays. 
```

{{% alert title="About using strings with JSON commands" color="warning" %}}
To specify a string as an array value to insert, wrap the quoted string with an additional set of single quotes. Example: `'"silver"'`. For more detailed use, see [Examples](#examples).
{{% /alert %}}
</details>

<details open><summary><code>index</code></summary> 

is position in the array where you want to insert a value. The index must be in the array's range. Inserting at `index` 0 prepends to the array. Negative index values start from the end of the array.
</details>

## Optional arguments

<details open><summary><code>path</code></summary> 

is JSONPath to specify. Default is root `$`.
</details>

## Return value 

> JSON.INSERT returns an array of integer replies for each path, the array's new size, or `nil`, if the matching JSON value is not an array. 

Reviewer: It makes sense to not add backticks when a link to the current doc would be generated,
However, backticks also affect formatting, allowing a common look for all of our commands in all pages.
(and a minor consideration: if copied elsewhere, would still be a valid link)

> JSON.INSERT returns an array of integer replies for each path, the array's new size, or `nil`, if the matching JSON value is not an array. 

Author: ```suggestion
`JSON.ARRINSERT` returns an array of integer replies for each path, the array's new size, or `nil`, if the matching JSON value is not an array. 
```

## Return value 

> `JSON.ARRINSERT` returns an array of integer replies for each path, the array's new size, or `nil`, if the matching JSON value is not an array. 

Reviewer: ```suggestion
`JSON.ARRINSERT` returns an [array](/docs/reference/protocol-spec/#resp-arrays) of integer replies for each path, the array's new size, or `nil`, if the matching JSON value is not an array. 
```
